src/rrt-viewer/RRTWidget.cpp

```diff
@@ -78,6 +78,14 @@ void RRTWidget::slot_setWaypointBias(int bias) {
     _biRRT->setWaypointBias((float)bias / 100.0f);
 }
 
+void RRTWidget::slot_setASC(int checked) {
+    if (checked == 0) {
```

Comment:
Just a minor note, but this block could be rewritten as `_biRRT->setASCEnabled(checked != 0)`.